pkg/security/apiserver/registry/podsecuritypolicysubjectreview/rest.go

```diff
@@ -107,10 +108,11 @@ var scheme = runtime.NewScheme()
 
 func init() {
 	utilruntime.Must(securityapi.Install(scheme))
+	utilruntime.Must(securityv1.Install(scheme))
```

Comment:
surprising to see in a registry. Why that?

Reply:
> surprising to see in a registry. Why that?

There's a helper function used below that identifies types for references.